app/background.cc

```diff
@@ -18,6 +20,7 @@ struct background_t {
     int frame;
     double fps;
     cv::Mat raw;
+    int bg_stored;
```

Comment:
`bool` perhaps as it's used only for true/false?

Reply:
You are right, I have modified this. Initializing was done with false, setting with true.